connector_phone_palitto_event_manager/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+
+from . import res_partner
+from . import res_company
```

Comment:
We're missing the User extension the methods to compute the cid and auth encrypted password